index.js

```diff
@@ -2787,6 +2787,28 @@
   }
   S.prop = def('prop', {a: [Accessible]}, [$.String, a, b], prop);
 
+  //# props :: Accessible a => Array String -> a -> b
+  //.
+  //. Takes an array of property names and an object with known properties and
+  //. returns the value of the specified property. If for some reason the
+  //. object lacks the specified property, a type error is thrown.
```

Comment:
This should be updated to reflect the fact that successive lookups are performed.